src/presenter/trip-list-presenter.js

```diff
@@ -2,24 +2,70 @@ import {render} from '../render.js';
 import TripListView from '../view/trip-list-view.js';
 import EditFormView from '../view/edit-form-view.js';
 import EventPointView from '../view/event-point-view.js';
+import {isPressEscape} from '../utils.js';
 
 export default class TripListPresenter {
-  tripListComponent = new TripListView();
+  #tripListComponent = new TripListView();
+
+  #tripListContainer = null;
+  #eventPointsModel = null;
+  #destinationModel = null;
+  #offerModel = null;
+
+  #eventPointsList = [];
+  #eventDestinations = [];
+  #eventOffersByType = [];
 
   init = (tripListContainer, eventPointsModel, destinationModel, offerModel) => {
-    this.tripListContainer = tripListContainer;
-    this.eventPointsModel = eventPointsModel;
-    this.destinationModel = destinationModel;
-    this.offerModel = offerModel;
-    this.eventPointsList = [...this.eventPointsModel.getEventPoints()];
-    this.eventDestinations = [...this.destinationModel.getDestination()];
-    this.eventOffersByType = [...this.offerModel.getOfferByType()];
-
-    render(new EditFormView(this.eventPointsList[0], this.eventDestinations, this.eventOffersByType), this.tripListComponent.getElement());
-    render(this.tripListComponent, this.tripListContainer);
-
-    this.eventPointsList.forEach((_, i) => {
-      render(new EventPointView(this.eventPointsList[i], this.eventDestinations, this.eventOffersByType), this.tripListComponent.getElement());
+    this.#tripListContainer = tripListContainer;
+    this.#eventPointsModel = eventPointsModel;
+    this.#destinationModel = destinationModel;
+    this.#offerModel = offerModel;
+    this.#eventPointsList = [...this.#eventPointsModel.eventPoints];
+    this.#eventDestinations = [...this.#destinationModel.destination];
+    this.#eventOffersByType = [...this.#offerModel.offerByType];
+
+    render(this.#tripListComponent, this.#tripListContainer);
+
+    this.#eventPointsList.forEach((event) => {
+      this.#renderEventPoints(event, this.#eventDestinations, this.#eventOffersByType);
     });
   };
+
+  #renderEventPoints = (eventPoints, destinations, offersByType) => {
+    const eventPointComponent = new EventPointView(eventPoints, destinations, offersByType);
+    const formEditComponent = new EditFormView(eventPoints, destinations, offersByType);
+
+    const replaceEventPointsListToEditForm = () => this.#tripListComponent.element.replaceChild(formEditComponent.element, eventPointComponent.element);
```

Comment:
Кажется можно подсократить без ущерба смыслу
```suggestion
    const replaceEventPointToEditForm = () => this.#tripListComponent.element.replaceChild(formEditComponent.element, eventPointComponent.element);
```